.github/workflows/master.yml

```diff
@@ -40,4 +40,14 @@ jobs:
       - name: Build and test Conformance-Gateway
         run: |
           cd Conformance-Gateway
-          mvn package -U -B -V --fail-at-end
+          mvn verify -U -B -V --fail-at-end
+
+      - name: Run SonarCloud analysis
+        env:
+          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }} # SonarCloud
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} # SonarCloud
+          BRANCH_NAME: ${{ github.head_ref || github.ref_name }}
+        run: |
+          cd Conformance-Gateway
+          mvn install -B -DskipTests
```

Comment:
Yes, that's true. We should avoid using `install` with maven, when possible. But according to the Sonar Maven scanner, for multi module projects, it was required to do `install` target before sonar is ran. 
Does it has consequences elsewhere? Because I figured, the used Docker image of a build is disposed anyways. Otherwise, I can do a short experiment without it.

Reply:
Update: I just tried a build without coverage. Looks fine, but is hard to tell, since my PR does not have many code changes. I'll keep it without `install` for now. If it does not work on normal PRs with Java code changed, I'll add it again.